mousestyles/data/__init__.py

```diff
@@ -62,3 +62,80 @@ def load_all_features():
     other_features = [x.iloc[:, -1] for x in data_frames]
     other_features.insert(0, all_features_df)
     return pd.concat(other_features, axis=1)
+
+
+def load_intervals(feature):
+    """
+    Parameters
+    ----------
+    feature: {"AS", "F", "IS", "M_AS", "M_IS", "W"}
```

Comment:
This should just be a **single** feature right i.e. **one** of the `{"AS", "F", "IS", "M_AS", "M_IS", "W"}` possible values. Also is this a _string input_ i.e. `"AS"` or a _set input_ `{"AS"}`